test/statetest/statetest_loader.cpp

```diff
@@ -140,6 +140,19 @@ inline uint64_t calculate_current_base_fee_eip1559(
     return base_fee;
 }
 
+namespace
+{
+bytes32 load_bytes32(const json::json& j)
```

Comment:
All things considered, I think we should just add this quirk to `from_json<hash256>` unconditionally. evmone is not test format sanitizer and probably never will be. So let's keep logic simple.

Reply:
OK